projects/whats_cooking_good_looking/whats_cooking_good_looking/train_ner_workflow.py

```diff
@@ -58,8 +57,7 @@ def evaluate_ner(labelstudio_tasks: bytes) -> dict:
     for ls_task in json.loads(labelstudio_tasks):
         annotation_result = ls_task["result"][0]["value"]
         for key in annotation_result:
-            if key == "id":
-                annotation_result.pop("id")
+            annotation_result.pop("id", None)
```

Comment:
Elle demandait plutôt un `annotation.pop("id", "Not found")` no ?

Reply:
Le 'not found' etait plutot pour mettre qqch dans l'autre cas je pense, mais le None est mieux imo, vu qu'on a pas besoin de retourner une string en soi